### ISSUE TITLE
Archive cloud status on compressed list view

### ISSUE THREAD

**Issue by cao-jacky** [mb1986/rm-hacks#338]
With the default views of Grid and List, there is a way to see if a document has been archived using reMarkable's Connect service, i.e., a cloud icon and the document preview is greyed out.  

Could such a thing be implemented to the compressed list view which was added in v0.0.8?

**Comment by atngames** [comment]
I cannot try this one, but could you please try this patch ?
``` bash -c "$(wget https://raw.githubusercontent.com/mb1986/rm-hacks/main/install.sh -O-)" _ patch 338-archived-on-compressed-list ```
Hopefuly, it does... somtething ?

**Comment by cao-jacky** [comment]
Yep it does, I have deployed it onto 3.11.2.5, and it works - that's great, thanks!

**Comment by atngames** [comment]
Awesome we will try to include this patch in the next release
